src/atoms/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Children, Component, ReactNode } from 'react';
+
+import styled from '_styled-components';
+import Typography from 'Typography';
+
+interface Props {
+  children: ReactNode;
+}
+
+export const TabsContainer = styled.ul`
+  display: flex;
+  text-align: center;
+  margin: 0;
+`;
+
+export const Tab = styled.li`
+  list-style: none;
+  flex: 1;
+  p {
+    margin: 0;
+  }
+`;
+
+export class Tabs extends Component<Props, {}> {
+  public render() {
+    const { children } = this.props;
+
+    return (
+      <TabsContainer>
+        {Children.map(children, child => (
+          <Tab>
+            <Typography>{child}</Typography>
+          </Tab>
+        ))}
+      </TabsContainer>
+    );
+  }
+}
+
+export default Tabs;
```

Comment:
I think we need a way to control the current tab, think of it more like a select menu or toggle